hack/e2e.sh

```diff
@@ -74,7 +74,7 @@ backoff oc -n openshift-monitoring \
 backoff oc apply -f /tmp/cluster-monitoring-config.yaml
 
 # Wait for user workload monitoring is deployed
-backoff oc -n openshift-user-workload-monitoring wait --for=condition=Ready pod -l app=thanos-ruler
+#backoff oc -n openshift-user-workload-monitoring wait --for=condition=Ready pod -l app=thanos-ruler
```

Comment:
It is not clear we need this.

Reply:
Investigating why UWM verification doesn't work as expected. I'll move this to a separate PR